homeassistant/components/upnp/__init__.py

```diff
@@ -195,14 +194,19 @@ async def async_unload_entry(hass: HomeAssistant, config_entry: ConfigEntry) ->
 
 
 @dataclass
-class UpnpSensorEntityDescription(
-    SensorEntityDescription, BinarySensorEntityDescription
-):
+class UpnpBinarySensorEntityDescription(EntityDescription):
```

Comment:
```suggestion
class UpnpBinarySensorEntityDescription(BinarySensorEntityDescription):
```
Will be missing an import

Reply:
This was pushed yesterday, not sure why you were seeing older version